docker-compose.test.yml

```
@@ -1,15 +1,23 @@
       dockerfile: Dockerfile
-    command: pnpm run start-prod
+    command: sh -c "npx next build && npx next-export-optimize-images && npx serve -s out -p 3001"
     restart: unless-stopped
     volumes:
       - .:/app
       - /app/node_modules
     ports:
       - '3001:3001'
+    healthcheck:
+      <<: *default-healthcheck
+      test: ["CMD","curl","-f","http://localhost:3001"]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)